src/components/AppController.tsx

```diff
@@ -144,13 +184,14 @@ const AppController = ({children}: AppControllerProps) => {
         logEventNum,
         logFileManagerProxy,
         numEvents,
+        setLogEventNum,
         setUiState,
         updatePageData,
     ]);
 
-    // On `filePath` update, load file.
+    // On `fileSrc` update, load file.
```

Comment:
I believe this useEffect hook can be removed completely

Reply:
see https://github.com/y-scope/yscope-log-viewer/pull/286/files#r2108403681 for the rationale